src/base/utils/string.cpp

```diff
@@ -119,23 +57,25 @@ Utils::String::NaturalCompare::NaturalCompare()
 #endif
 }
 
-bool Utils::String::NaturalCompare::operator()(const QString &l, const QString &r)
+bool Utils::String::NaturalCompare::operator()(const QString &left, const QString &right)
 {
+    // case-insensitive comparison
```

Comment:
Sorry, I don't remember why I rejected it.
>and why the idea now?

You are replacing the old algorithm here. And unless I am mistaken it wasn't case-insensitive...